### PR TITLE
Windows Compilation

### PR DESCRIPTION
**Pull request by deslaughter**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by deslaughter** [comment]
@conda-forge-admin, please rerender

**Review by gbarter — APPROVED** [review]
Let me know if I can help with the debugging

File: recipe/meta.yaml [hunk 1]
```diff
@@ -6,33 +6,31 @@ package:
...
 requirements:
   build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - {{ compiler('fortran') }}
-    - {{ compiler('m2w64_c') }}  # [win]
-    - {{ compiler('m2w64_cxx') }}  # [win]
-    - make
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('cxx') }}  # [not win]
```
Comment (hmaarrfk) [inline recipe/meta.yaml:21]: why did you all remove the compilers from windows? this doesn't seem right...

File: recipe/meta.yaml [hunk 2]
```diff
@@ -6,33 +6,31 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/openfast/openfast/archive/v{{ version }}.tar.gz
-  sha256: 94da085f1d8861a93ab32b6b39335a403d182be0b5f7ff48f63d5805eb7f2b2c
```
Comment (hmaarrfk) [inline recipe/meta.yaml:10]: why would you remove the usage of the url and sha256? this is the best practice at conda forge.

**Pull request merged**